pkg/ipamd/ipamd.go

```diff
@@ -138,9 +140,6 @@ const (
 	// vpcENIConfigLabel is used by the VPC resource controller to pick the right ENI config.
 	vpcENIConfigLabel = "vpc.amazonaws.com/eniConfig"
```

Comment:
Even this can be cleaned up right?

Reply:
Cleaning work has been done in the latest update.